spec/amp-actions-and-events.md

```diff
@@ -666,8 +666,8 @@ actions that apply to the whole document.
     <td>Tries to close the window if allowed, otherwise it navigates similar to <code>navigateTo</code> Action. Useful for use-cases where a "Back" button may need to close the window if it were opened in a new window from previous page or navigate if it wasn't opened.</td>
   </tr>
   <tr>
-    <td><code>goBack</code></td>
-    <td>Navigates back in history.</td>
+    <td><code>goBack(navigate=BOOLEAN)</code></td>
+    <td>When navigate is false or unspecified, goBack will attempt to go back in history while remaining in the current document. When navigate is true, the action will allow for cross-document navigation, similar to [history.back](https://developer.mozilla.org/en-US/docs/Web/API/History/back).</td>
```

Comment:
I think `goBack` does navigate backwards across pages on publisher origin.

Reply:
I would also put `navigate`, `false`, and `goBack` in code font.
And I'd change "allow for" to "allow" - since it simply directly allows cross-document navigation!

otherwise LGTM!